src/validate-options.ts

```diff
@@ -0,0 +1,48 @@
+import { HoneycombOptions } from './types';
+import {
+  isClassic,
+  MISSING_API_KEY_ERROR,
+  MISSING_SERVICE_NAME_ERROR,
+} from './util';
+
+export const IGNORED_DATASET_ERROR =
+  '🔕 WARN: Dataset is ignored in favor of service name.';
+export const MISSING_DATASET_ERROR =
+  '❌ WARN: Missing dataset. Specify either HONEYCOMB_DATASET environment variable or dataset in the options parameter.';
+export const SKIPPING_OPTIONS_VALIDATION_MSG =
+  '⏭️ DEBUG: Skipping options validation. To re-enable, set skipOptionsValidation option or HONEYCOMB_SKIP_OPTIONS_VALIDATION to false.';
+export const SAMPLER_OVERRIDE_WARNING =
+  '🔨 WARN: Default deterministic sampler has been overridden. Honeycomb requires a resource attribute called SampleRate to properly show weighted values. Non-deterministic sampleRate could lead to missing spans in Honeycomb. See our docs for more details. https://docs.honeycomb.io/getting-data-in/opentelemetry/node-distro/#sampling-without-the-honeycomb-sdk';
+
+export const validateOptionsWarnings = (options?: HoneycombOptions) => {
+  if (options?.skipOptionsValidation) {
```

Comment:
Decided to do the logic of skipping validation inside the function instead at the top level in the SDK because it meant we could keep the `SKIPPING_OPTIONS_VALIDATION_MSG` contained to this file